### PR TITLE
[16.0][MIG] stock_picking_quick : Migration to 16.0

### PR DESCRIPTION
**Pull request by FranzPoize**

### CODE REVIEW

**Comment by legalsylvain** [comment]
Hi @FranzPoize. what is the state of the PR ? 

thanks !

**Comment by FranzPoize** [comment]
Hi @legalsylvain, I'm still waiting on https://github.com/OCA/product-attribute/pull/1393 if it's not done after the new year i'll make a new one and we'll be able to move forward with this

**Comment by FranzPoize** [comment]
Hi @legalsylvain If you want to take a look this that would be awesome !

**Comment by bealdav** [comment]
Here Add button is not visible whereas you may add a line. 

image

Then that's not ok for me

File: stock_picking_quick/views/picking_view.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    <record id="view_picking_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <field name="move_ids_without_package" position="before">
+                <button
+                    name="add_product"
+                    string="Add"
+                    type="object"
+                    attrs="{'invisible': [('state', 'not in', ('draft', 'waiting'))]}"
```
Comment (bealdav) [inline stock_picking_quick/views/picking_view.xml]: assigned state should support visible button

**Comment by rousseldenis** [comment]
@FranzPoize Could you rearrange your commits in order to follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-16.0#technical-method-to-migrate-a-module-from-150-to-160-branch ? Thanks

File: stock_picking_quick/views/picking_view.xml (hunk 1 above)
Comment (FranzPoize) [reply stock_picking_quick/views/picking_view.xml]: Should confirmed state also display the button ?

Comment (bealdav) [reply stock_picking_quick/views/picking_view.xml]: `ADD` button should be visible in the same conditions than `Add a line` I suppose, then also confirmed. 

cc @Kev-Roche @sebastienbeau  please could give your point of view here ?

**Comment by FranzPoize** [comment]
@bealdav @rousseldenis Should be all good now. Thank you for your feedback

**Review by bealdav — APPROVED** [review]
LGTM, thanks a lot

**Review by rousseldenis — APPROVED** [review]
LGTM

**Pull request merged**